lib/appscale_info.py

```diff
@@ -38,6 +38,16 @@ def get_keyname():
   """ Returns the keyname for this deployment. """
   return get_db_info()[':keyname']
 
+def get_all_ips():
```

Comment:
@menivaitsi what do you think about pulling the IPs from the appcontroller from the current layout? It seems a bit of a safer way for me ...